securecookie.go

```diff
@@ -123,6 +125,16 @@ func (s *SecureCookie) BlockFunc(f func([]byte) (cipher.Block, error)) *SecureCo
 	return s
 }
 
+// RandomReader sets the reader to use when generating random data.
+// The default should only be changed if the given reader can provide
+// cryptographically secure random data.
+//
+// Default is crypto/rand.Reader.
+func (s *SecureCookie) RandomReader(randomReader io.Reader) *SecureCookie {
+	s.randomReader = randomReader
```

Comment:
What happens if someone sets `s.randomReader` while another goroutine tries to use it?